### PR TITLE
docs: update the README

### PR DESCRIPTION
**Pull request by moldy530**
## PR-Codex overview
This PR focuses on updating the dependencies in the `package.json` files and removing unnecessary code. 

### Detailed summary
- Added `@alchemy/aa-core` as a dependency in the `packages/alchemy/package.json` and `packages/accounts/package.json` files.
- Removed `@alchemy/aa-core` as a devDependency in the `packages/ethers/package.json` file.
- Removed unnecessary code and comments in the `README.md` file.

> The following files were skipped due to too many changes: `README.md`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: README.md
Comment (avasisht23) [inline README.md:1]: went to your branch and saw ::code-group style is off:

image

File: README.md [hunk 1]
```diff
@@ -1,241 +1,32 @@
 # Account Abstraction SDK (aa-sdk)
 
-⚠️ This repo is actively being developed and certain features might not be fully implemented yet or are subject to change ⚠️
+The `aa-sdk` is a type-safe and performant TypeScript library built on top of [viem](https://viem.sh/) to provide ergonomic methods for sending user operations, sponsoring gas, and deploying smart contract accounts. It handles all the complexity of ERC-4337 under the hood to make account abstraction simple.
```
Comment (avasisht23) [inline README.md:3]: can you add a clear CTA and/or callout section that inks to the docs landing page?

File: README.md
Comment (moldy530) [reply README.md:1]: fixed it

File: README.md (hunk 1 above)
Comment (moldy530) [reply README.md:3]: Fixed it

**Review by avasisht23 — APPROVED** [review]

**Pull request merged**